### ISSUE TITLE
Latest 1.7 - Scheduler jobs are duplicated

### ISSUE THREAD

**Issue by acondura** [getgrav/grav#3081]
Strangest thing, all custom jobs are duplicated but not the default ones. They are listed twice in the list and run twice.
Jobs are added in onSchedulerInitialized, I checked and rechecked, there's no duplication in my custom plugin code. I looked through the Grav scheduler core code and tried debugging but couldn't figure it out where it's coming from.

<img width="1684" alt="Screenshot 2020-11-25 at 14 54 32" src="https://user-images.githubusercontent.com/1694890/100230342-4bb3ae80-2f2e-11eb-924f-60c39e1eff2d.png">

**Comment by rhukster** [comment]
![rhuk@hades:~workspacegrav-sandbox 2020-12-10 at 11 21 52 AM](https://user-images.githubusercontent.com/1084697/101813418-f0f78680-3ad9-11eb-9a01-18a39c063d18.jpg)

I can't replicate this.. It must be something related to your installation, some plugin or something?

**Comment by acondura** [comment]
In my case, I've defined custom jobs in the onSchedulerInitialized event not through the admin UI.

![Screenshot from 2020-12-12 16-09-21](https://user-images.githubusercontent.com/1694890/101986105-96565a00-3c94-11eb-9d5d-a827feccd4c5.png)

**Comment by acondura** [comment]
I've updated to the latest 1.7-rc.19 and the duplicates went away.

![Screenshot from 2020-12-13 01-18-35](https://user-images.githubusercontent.com/1694890/101999020-579ac100-3ce1-11eb-88a2-7c24cc6a5320.png)

The errors I get now are in the form of:
sh: exec: line 1: bin/plugin fetchdrupal getBlocks: not found

**Comment by rhukster** [comment]
Ok, sounds like the error you see now is related to your `fetchdrupal` plugin?  What does the CLI command look like?

**Comment by acondura** [comment]
Seems to work fine from terminal.
![Screenshot from 2020-12-13 13-21-36](https://user-images.githubusercontent.com/1694890/102017224-3af89a80-3d6e-11eb-9d56-2a0e1977ff86.png)

Running manually gives error:
![Screenshot from 2020-12-13 13-24-26](https://user-images.githubusercontent.com/1694890/102017232-49df4d00-3d6e-11eb-800f-a22e52ec0f1d.png)

and the CLI command:
![Screenshot from 2020-12-13 13-22-11](https://user-images.githubusercontent.com/1694890/102017238-5a8fc300-3d6e-11eb-8ec0-7c61d0428794.png)

**Comment by acondura** [comment]
In onSchedulerInitialized, changing this:
$job = $scheduler->**addCommand**('bin/plugin fetchdrupal getBlocks', [], 'fetchdrupal-blocks');

to this (like I used to have in the very beginning):
$job = $scheduler->**addFunction**('Grav\Plugin\FetchDrupalPlugin::getBlocks', [], 'fetchdrupal-blocks');

solved it. At least, it works manually. I now have to get cron working in the container.

**Comment by acondura** [comment]
I did get cron working but not with the default way. I use an Alpine container and had to use Supercronic (https://github.com/aptible/supercronic) because I ran into issues when runinng cron jobs by non root users. Full details here:
https://stackoverflow.com/questions/44934762/why-is-crond-failing-to-run-a-non-root-crontab-on-alpine-linux.
Maybe Supercronic could be taken into account when checking for a crontab in /system/src/Grav/Common/Scheduler/Scheduler.php line 299 under isCrontabSetup().
